src/lib/expansion/expansion-animations.ts

```diff
@@ -35,12 +35,12 @@ export const matExpansionAnimations: {
     state('collapsed', style({
       height: '{{collapsedHeight}}',
     }), {
-      params: {collapsedHeight: '48px'},
+      params: {collapsedHeight: '*'},
     }),
     state('expanded', style({
       height: '{{expandedHeight}}'
     }), {
-      params: {expandedHeight: '64px'}
+      params: {expandedHeight: '*'}
```

Comment:
I agree that it's a better way to handle custom height, but AFAIK it will break the people that are using the binding as it's set up right now. We could either remove the inputs and leave these changes for a later release, or deprecate them and set up the bindings so they just set the height as an inline style.

Reply:
It shouldn't break those using it as it is currently since we define the same expanded and collapsed heights (64px and 48px) in css rather than the animation parameters.

If a user set it by changing the animation params, it should be respected as it will be put into the style tag and override our css rule's height.
If a user set it by setting the params to `*` and defined the css rule's, then it results as the same how the new set up works.

Am I missing a scenario that would break?

Reply:
Sorry, I probably got confused last time. Looking at it again, it should be fine.